modules/sdk-coin-eth/test/unit/transactionBuilder/flushTokens.ts

```diff
@@ -182,10 +183,12 @@ describe('Eth Transaction builder flush tokens', function () {
         forwarderVersion: 2,
       });
       const txBuiderFromRaw: any = getBuilder('teth');
-      txBuiderFromRaw.fromImplementation(tx.toBroadcastFormat());
-      // txBuiderFromRaw.type.should.equal(TransactionType.FlushTokens);
-      txBuiderFromRaw._forwarderAddress.should.equal('0x53b8e91bb3b8f618b5f01004ef108f134f219573');
```

Comment:
Why these cases where removed for < 4?, is it constitute a breaking change?

Reply:
It is not removed. It is implicit in

`      txBuiderFromRaw.from(tx.toBroadcastFormat());
      const result = await   ##txBuiderFromRaw.build();`